pandas/core/frame.py

```diff
@@ -3889,7 +3885,6 @@ def insert(self, loc, column, value, allow_duplicates: bool = False) -> None:
                 "'self.flags.allows_duplicate_labels' is False."
             )
         value = self._sanitize_column(value)
-        value = _maybe_atleast_2d(value)
```

Comment:
i think this removed all the usages of maybe_atleast_2d, so can remove the function

Reply:
Indeed, will remove that in one of my next PRs